apps/admin-ui/cypress/e2e/client_authorization_test.spec.ts

```diff
@@ -68,6 +69,22 @@ describe("Client authentication subtab", () => {
     authenticationTab.formUtils().cancel();
   });
 
+  it("Edit a resource", () => {
+    authenticationTab.goToResourcesSubTab();
+    listingPage.goToItemDetails("Resource");
+
+    new CreateResourcePage()
+      .fillResourceForm({
+        displayName: "updated",
+      })
+      .formUtils()
+      .save();
+
+    masthead.checkNotificationMessage("Resource successfully updated", true);
```

Comment:
I believe `true` is the default here, so it can be omitted.
```suggestion
    masthead.checkNotificationMessage("Resource successfully updated");
```